docs/plus/index.md

```diff
@@ -164,7 +164,7 @@ app.synth();
 
 === "Python"
 
-    `❯ pip install cdk8s-plus-17 cdk8s`
```

Comment:
How will we remember to change this when we graduate?

Let's open a 1.0.0 release GitHub issue and add to a checklist?

Reply:
yeap. I'll just create the milestone and add it as an issue.